src/index.tsx

```diff
@@ -38,6 +38,7 @@ const IndexPageWrapper = () => {
                 </a>
                 .
             </p>
+            <style>{`body { padding: 0 10px;}`}</style>
```

Comment:
This necessary to avoid the overflow or just an additional unrelated change?

Reply:
Yeah, unrelated, the index.tsx is used *just* for local testing (it doesn't get bundled for the app)